libbeat/logp/config.go

```diff
@@ -24,18 +24,18 @@ import (
 // Config contains the configuration options for the logger. To create a Config
 // from a common.Config use logp/config.Build.
 type Config struct {
-	Beat       string   `config:",ignore"`   // Name of the Beat (for default file name).
-	JSON       bool     `config:"json"`      // Write logs as JSON.
-	Level      Level    `config:"level"`     // Logging level (error, warning, info, debug).
-	Selectors  []string `config:"selectors"` // Selectors for debug level logging.
-	ECSEnabled bool     `config:"ecs"`       // Adds minimal ECS information using ECS conformant keys to every log line
```

Comment:
Could this on by default?

Reply:
ECS is on by default here https://github.com/elastic/beats/pull/19211/files#diff-7b5f5c8d014c26b7f6c249cb83fb54b9R101